### PR TITLE
front: change imported train schedules number of step color

### LINKED ISSUE #9594: Viriato import with 0 OP matched
### What happened?

From a Viriato file import, if a train has no OCP matched with an OP (= we do not match every OP in our infra), the import is should not work.

See all "-1":
Image

I also break the Macro mode, but it is rather a consequence that will be solved by the above bug

Linked to this [bug](https://github.com/OpenRailAssociation/osrd/issues/9610)

### What did you expect to happen?

When no stop is matched for a train, simply do not import it and console log it

### How can we reproduce it (as minimally and precisely as possible)?

Download the Viriato file ("MSC-AIX...") from this [folder](https://sncf.sharepoint.com/:f:/r/sites/OSRD644GrpO365/Documents%20partages/Donn%C3%A9es/3-Fichiers%20Viriato?csf=1&web=1&e=tsLWf3)

line 17 of this file, we see the OCP that is not match in our infra, used by the `trainPart id="tp_3_1975735"`

- import in operational studies
- notice 

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

dev 95c1305

### PR DESCRIPTION
**Pull request by RomainValls**
closes https://github.com/osrd-project/osrd-confidential/issues/884
Will close #9594

### CODE REVIEW

File: front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainDetail.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default function ImportTrainScheduleTrainDetail({
         </span>
         <span
           className={`import-train-schedule-traindetail-stepnb ${
-            trainData.steps.length <= 2 ? 'bg-secondary' : 'bg-primary'
+            trainData.steps.length <= 2 ? 'bg-danger' : 'bg-primary'
```
Comment (theocrsb) [inline front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainDetail.tsx]: Maybe we should use a variable in our color variables? like `error`?

Comment (clarani) [reply front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainDetail.tsx]: Ah yes you're right, I thought danger and error60 were the same but they are not. Can you use error60 (you can see it in our `styles/_variables.scss`

Comment (RomainValls) [reply front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainDetail.tsx]: that's better, thanks !

**Review by theocrsb — APPROVED** [review]
LGTM (not tested)

**Review by louisgreiner — APPROVED** [review]
LGTM, tested, thanks

image

File: front/src/modules/trainschedule/styles/ImportTrainSchedule/_importTrainScheduleTrainsList.scss [hunk 2]
```diff
@@ -86,6 +86,9 @@
           height: 32px;
           text-align: center;
         }
+        .import-invalid-stepnb {
```
Comment (clarani) [inline front/src/modules/trainschedule/styles/ImportTrainSchedule/_importTrainScheduleTrainsList.scss]: ;)
```suggestion
        .import-invalid-steps-nb {
```

**Review by clarani — APPROVED** [review]
Left a small comment, but LGTM except that ! :rocket:

**Pull request merged**